Lib/test/test_funcattrs.py

```diff
@@ -478,6 +479,27 @@ def test_builtin__qualname__(self):
         self.assertEqual([1, 2, 3].append.__qualname__, 'list.append')
         self.assertEqual({'foo': 'bar'}.pop.__qualname__, 'dict.pop')
 
+    @support.cpython_only   # See gh-58211
+    def test_builtin__self__(self):
```

Comment:
```suggestion
    @support.cpython_only
    def test_builtin__self__(self):
        # See https://github.com/python/cpython/issues/58211.
```

Reply:
If this is not sufficient, a summary of the discussion can be added (but not using docstrings as it would be present in case of failure).